src/main.js

```diff
@@ -1 +1,12 @@
-console.log("Car");
+console.log('Car');
+
+class Car {
+  constructor(name) {
+    this.name = name;
+    this.position = 0;
+  }
+
+  moveForward() {
+    this.position += 1;
+  }
+}
```

Comment:
```suggestion
class Car {
  position = 0;
  name;
  
  constructor(name) {
    this.name = name;
  }

  moveForward() {
    this.position += 1;
  }
}
```

이렇게 class field에 멤버변수를 정의할 수도 있답니다 ㅎㅎ